configure.ac

```diff
@@ -161,6 +161,20 @@ AS_IF([test "x${RKT_RUN_FUNCTIONAL_TESTS}" = 'xyes' -o "x${RKT_RUN_FUNCTIONAL_TE
 
 AC_SUBST(RKT_RUN_FUNCTIONAL_TESTS)
 
+AS_VAR_IF([cross_compiling], [no], [GOARCH_FOR_BUILD=${GOARCH_FOR_BUILD:-${GOARCH}}],
```

Comment:
Maybe add the comment about `cross_compiling` variable that it is set by autoconf itself.